README.md

```diff
@@ -40,10 +40,18 @@ when this project reaches a stable state.
 
 ## Generating test files
 
+Setup Environment :-
+
+    - Install postgres, 
```

Comment:
Just a couple small things:

1) Up above there's a `Setup` section. I would just add a dependency on Postgres v10.6 (the version I'm coding against). You can add the instructions on how to install that particular version there.
2) In this section, I would just mention in a sentence or two that the user should make sure he/she has the postgres server running so that `psql` is able to run against each test query. You can keep the instruction on how to start the server if you'd like. Thanks for that.